shark/shark_benchmark_runner.py

```diff
@@ -132,12 +132,12 @@ def benchmark_torch(self, modelname):
         frontend_model.to(torch_device)
         input.to(torch_device)
 
-        try:
-            frontend_model = torch.compile(
-                frontend_model, mode="max-autotune", backend="inductor"
-            )
-        except RuntimeError:
-            frontend_model = HFmodel.model
+        # try:
```

Comment:
dead code?

Reply:
Will need to re-enable as soon as CUDA context issues from running inductor in a forked subprocess are resolved.

Reply:
Ideally add a comment to that effect